### PR TITLE
Update DG by adding Workload Distribution Feature

### PR DESCRIPTION
**Pull request by QY-H00**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -191,6 +191,30 @@ The notdone mechanism is similar to that of the done mechanism, except that the
 
 <div style="page-break-after: always;"></div>
 
+### Workload Distribution feature
+
+#### Implementation
+
+The workload distribution feature is implemented by `ModuleManager`. It uses HashMap and an OberservableList to monitor the workload of each module and PieChart in javaFX to visualizes it. More explicitly, it implements by 4 HashMaps using Module as key. The first three maps are used in the ModuleList while the map of weighted sum is used in the PieChart.
```
Comment (yhtMinceraft1010X) [inline docs/DeveloperGuide.md]: I think instead of "it implements by 4 HashMaps", it should be "it is implemented by 4 HashMaps".

**Review by yhtMinceraft1010X — COMMENTED** [review]
Generally LGTM

**Review by figo2127 — COMMENTED** [review]
LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (QY-H00) [reply docs/DeveloperGuide.md]: OK!

**Pull request merged**